bindata/network/ovn-kubernetes/managed/ovnkube-control-plane.yaml

```diff
@@ -152,7 +152,7 @@ spec:
             --config-file=/run/ovnkube-config/ovnkube.conf \
             --k8s-token-file=/var/run/secrets/hosted_cluster/token \
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
-            --metrics-bind-address "127.0.0.1:9108" \
+            --metrics-bind-address "0.0.0.0:9108" \
```

Comment:
what is the reason behind changing this from localhost to making it listen at all IPs on the pod? This pod is CNI networked and not hostnetworked right?

Reply:
`127.0.0.1` is wrong, there is a service exposing these ports from the pods.
This bug got introduced during IC refactor: https://github.com/openshift/cluster-network-operator/commit/62d8b3d69d8a80b461e708c4108aac6170242b2e#diff-847acb60cc87b24725974ffa2f282281351cf4214bf98bc5bfcf58c4763f1d30L887
You can see that the single-zone still has the address bound to 0.0.0.0:
https://github.com/openshift/cluster-network-operator/blob/62d8b3d69d8a80b461e708c4108aac6170242b2e/bindata/network/ovn-kubernetes/managed/single-zone-interconnect/ovnkube-master.yaml#L889

Reply:
Yes, In managed cluster kube-rbac-proxy is not there to scrape metrics from ovnkube-cluster-manager. Instead metrics endpoint of ovnkube-cluster-manager gets exposed directly.  
ovnkube-control-plane of hosted cluster does run on ClusterNetwork and not on hostnetwork.

Reply:
Will this bind correctly on IPv6 single-stack?

Reply:
Yep. `0.0.0.0` should work on IPv6. https://github.com/golang/go/issues/48723#issuecomment-932829647

Reply:
Thanks @rbbratta for raising this question and clarifying it by yourself only. :)